src/rust/Cargo.toml

```
@@ -1,10 +1,11 @@
 [package]
 name = 'helloextendr'
-version = '0.2.0'
-edition = '2018'
+version = '0.1.0'
```

One last thing. Should we perhaps increment this version rather than resetting it to default? `0.3.0` seems appropriate, don't you think?

---

I'll add a suggestion so you can apply it directly from here if you agree
```suggestion
version = '0.3.0'
```

---

Just a question: at the moment I think the version of this file is still 0.1.0 unless it is updated manually, should it be updated in general?
If so, it would be useful to be able to update it along with the R package version with a function like `usethis::use_version`.

---

That's worth discussing, @eitsupi could you perhaps file an issue under `rextendr`?